source/common/access_log/access_log_formatter.cc

```diff
@@ -275,6 +278,9 @@ std::vector<FormatterProviderPtr> AccessLogFormatParser::parse(const std::string
         const std::string args = token[StartTimeParamStart - 1] == '('
                                      ? token.substr(StartTimeParamStart, parameters_end)
                                      : "";
+        if (std::regex_search(args, NEWLINEPATTERN)) {
```

Comment:
Can you add a comment explaining that the formatted string may be destined for headers, where we want to avoid the possibility of {NUL, LR, CF}, hence we validate the input specifier here.